src/features/Tiles/MovieTiles/index.js

```diff
@@ -12,14 +15,13 @@ import {
 } from "./styled";
 import { Rate } from "../../Rate";
 
+const MovieTiles = ({ title, movieData }) => {
```

Comment:
movieData is probably not needed here. You can get all the necessary info from popularMoviesData i suppose... 🤔